crates/indexer/src/processors/default_processor.rs

```diff
@@ -460,15 +522,29 @@ impl TransactionProcessor for DefaultTransactionProcessor {
                 },
             }
         }
+
+        // TODO, merge this loop with above
+        let mut all_objects = vec![];
+        let mut all_current_objects = HashMap::new();
+        for txn in &transactions {
+            let (mut objects, current_objects) = Object::from_transaction(txn);
+            all_objects.append(&mut objects);
+            all_current_objects.extend(current_objects);
+        }
         // Getting list of values and sorting by pk in order to avoid postgres deadlock since we're doing multi threaded db writes
         let mut current_table_items = current_table_items
             .into_values()
             .collect::<Vec<CurrentTableItem>>();
         let mut table_metadata = table_metadata.into_values().collect::<Vec<TableMetadata>>();
+        let mut all_current_objects = all_current_objects
+            .into_values()
+            .collect::<Vec<CurrentObject>>();
```

Comment:
I'm curious, why do you make a hashmap of objects, and then drop the keys?  Do you get duplicates?

Reply:
This is current objects across a list of transactions.